src/lib.rs

```diff
@@ -220,6 +220,9 @@ pub(crate) use self::{
 #[cfg(feature = "mock_crypto")]
 pub(crate) use self::mock::crypto;
 
+#[cfg(any(test, feature = "mock_base"))]
+pub(crate) use unwrap::unwrap;
```

Comment:
better put a comment here to explain why we do so.
so that we don't `change it back` later on accidently.

Reply:
I'm thinking someone wanting to change it back would look at the history of why it is this way, or the reviewers would question it.